tests/testthat/test-ppm.R

```diff
@@ -31,11 +31,6 @@ test_that("ppm_snapshots", {
     `2023-02-27T00:00:00Z` = "17028146",
     `2023-02-28T00:00:00Z` = "17054670"
   )
-  mockery::stub(
-    ppm_snapshots,
-    "async_get_ppm_versions",
-    function(...) async_constant(ver)
-  )
```

Comment:
I don't understand what this one is supposed to do. I don't see `async_get_ppm_versions` defined anywhere. Was this a leftover (that mockery didn't complain about)?